src/error.rs

```diff
@@ -227,26 +231,6 @@ impl From<std::io::ErrorKind> for ErrorKind {
     }
 }
 
-impl From<trust_dns_resolver::error::ResolveError> for ErrorKind {
```

Comment:
Just pushed a commit to remove these which I missed in the first pass. Thankfully the [Rust API Guidelines](https://rust-lang.github.io/api-guidelines/necessities.html#public-dependencies-of-a-stable-crate-are-stable-c-stable) called this case out specifically since it was so easy to miss.